<?xml version="1.0" encoding="UTF-8" standalone="no"?><testsuite errors="0" failures="1" name="Happy path test" skipped="0" tests="1" time="17.17" timestamp="2025-08-29 04:15:57"><testcase name="As a user over 30 I want to check if I am exempt from paying for prescriptions"><failure message="Unexpected message content. Actual: &quot;we cannot match you to our records&quot;" type="java.lang.AssertionError">Unexpected message content. Actual: "we cannot match you to our records"</failure><system-err>Unexpected message content. Actual: "we cannot match you to our records"
nhsbsa.exch.pages.ResultsPage.verifyNoMatchMessageIsDisplayed(ResultsPage.java:46)
nhsbsa.exch.action.ResultsPageAction.I_verify_the_no_match_message_is_displayed(ResultsPageAction.java:19)
nhsbsa.exch.stepDefinitions.HomePageStepDefs.i_should_see_we_cannot_match_you_to_our_records(HomePageStepDefs.java:69)
</system-err></testcase></testsuite>